test/strategy/ExtendConfigStrategyTest.js

```diff
@@ -7,6 +7,43 @@ var strategy = require('../../lib/strategy');
 var ExtendConfigStrategy = strategy.ExtendConfigStrategy;
 
 describe('ExtendConfigStrategy', function () {
+  it("should be able to extend member that references itself", function (done) {
+    var extendConfig = {
+      abc: "123"
```

Comment:
Personally I prefer unique values that are generated at runtime, as I love the idea of adding chaos to my tests and still having them survive :)

But the workflow of needing to search for the strings is also important.  Review approved!